README.md

```diff
@@ -21,44 +21,22 @@ To install this package from PyPi, run
 pip install quasielasticbayes
 ```
 
-## Develop: Building the Conda package
-
-To create and activate the build environment, run
-
-```sh
-mamba env create -f conda/conda-build-py310.yml && conda activate qeb-conda-build-py310
-```
-
-To build a Conda package, run
-
-```sh
-conda build conda/recipe/ --output-folder .
-```
-
-At the end of the conda build step, the tests are run automatically to verify the package is working.
-
 ## Develop: Building the PyPi package
```

Comment:
Could we change the title of this section to something like `## Develop: Building & testing locally`